gnpy/core/convert.py

```diff
@@ -155,7 +155,7 @@ def convert_file(input_filename, filter_region=[]):
                                         'longitude': x.longitude}},
               'type': 'Fused'}
              for x in nodes_by_city.values() if x.node_type.lower() == 'fused'] +
-            [{'uid': f'fiber ({x.from_city} → {x.to_city})-{x.east_cable}',
+            [{'uid': f'fiber ({x.from_city} \u2192 {x.to_city})-{x.east_cable}',
```

Comment:
I don't think this file needs to be changed.

Reply:
OK for me as long as the generated uid can be used in excel sheet route constraint. I checked that it was working with this code before you made the utf8 modif...

Reply:
We can change the identifiers if we'd like, but one advantage of keeping this Unicode character is that it will suss out other internationalisation issues. 

We don't want everything to work smoothly for "Lyon" then blow up on "Rhône"!

Reply:
OK !